.github/workflows/periodic.yml

```diff
@@ -57,6 +57,27 @@ jobs:
       test-matrix: ${{ needs.linux-bionic-cuda11_6-py3-gcc7-slow-gradcheck-build.outputs.test-matrix }}
       timeout-minutes: 300
 
+  linux-bionic-cuda11_6-py3_10-gcc7-periodic-dynamo-benchmarks-build:
```

Comment:
Can we re-use the build from another workflow? Or do you build this one in a specific way?

Reply:
Hmm, so it looks like the gradcheck build is nearly equivalent, the only problem is it's not being built for the correct cuda arch. However, I actually want to move this into its own workflow so it can be triggered by itself (without the rest of periodic); to do that, it would need a dedicated build. In any case, this is periodic, so it's probably not too bad?

Reply:
If you don't have the right arch that sounds good.
I was just asking in case a one line change would avoid a full cuda build.